Integrations/integration-ArcSightLogger.yml

```diff
@@ -634,20 +577,14 @@ script:
       description: Indicates that the search should try to discover fields in the
         events found.
     - name: startTime
-      description: 'The date and time string for the start time in the search time range.
-        If start_ time is provided, end_time must be present as well. 
-        Format: 
-        - InPDT:2014-05-26T21:49:46.000-07:00
-        - InUTC:2014-05-26T21:49:46.000Z
-        '
+      description: 'The date and time string for the start time in the search time
+        range. If start_ time is provided, end_time must be present as well. Format:
```

Comment:
start_time => startTime
end_time => endTime

Reply:
note that args.start_time and args.end_time are passed to the functions.
will fix it.